modules/controllers/jobs-controller.ts

```diff
@@ -162,15 +168,15 @@ export function JobsController(tracer?: any) {
             });
             const dbIds = pools.map((pool) => pool.id.toLowerCase());
             const viemClient = getViemClient(chain);
+            const vaultClient = getVaultClient(viemClient, vaultAddress);
 
             const { changedPools } = await getChangedPools(vaultAddress, viemClient, BigInt(fromBlock));
             const ids = changedPools.filter((id) => dbIds.includes(id.toLowerCase())); // only sync pools that are in the database
             if (ids.length === 0) {
                 return [];
             }
-            await syncPools(ids, viemClient, vaultAddress, chain);
-            await syncTokenPairs(ids, viemClient, routerAddress, chain);
```

Comment:
still needs to be synced somewhere and we gotta make sure it's synced AFTER the pools are synced as it needs updated balances in the DB

Reply:
removed by accident when merging conflicts